### ISSUE TITLE
Github - master is not correct

### ISSUE THREAD

**Issue by SteveDoyle2** [rtfd/readthedocs.org#1964]
Per the latest RTD build:

https://readthedocs.org/projects/pynastran-git/builds/3678657/

The git hash is:
latest (1b5b697fc3f25162f483b347ea17fdc6019ac435)

The error is:

    File "/home/docs/checkouts/readthedocs.org/user_builds/pynastran-git/envs/latest/local/lib/python2.7/site-packages/pyNastran/op4/test_loadop4.py", line 25, in <module>
    matrices = op4.read_op4(in_file)
    File "/home/docs/checkouts/readthedocs.org/user_builds/pynastran-git/envs/latest/local/lib/python2.7/site-packages/pyNastran/op4/op4.py", line 103, in read_op4
    raise IOError('cannot find op4_filename=%r' % op4_filename)
    IOError: cannot find op4_filename='/home/docs/checkouts/readthedocs.org/user_builds/pynastran-git/envs/latest/local/lib/python2.7/site-packages/pyNastran/op4/test/mat_b_dn.op4'

In order to fix this bug, I decided to disable tests for readthedocs.  As this file was old, I deleted it.  As such, the file ``test_loadop4.py`` does not exist in the latest version.  The latest version (as of right now), which is pushed to the master is:
https://github.com/SteveDoyle2/pyNastran/commit/42403b18d1b978495c6593be8ea06de3dcecbaee

not:
https://github.com/SteveDoyle2/pyNastran/commit/1b0ae14dc83ba1d603e80591cc69344e2cba2568

**That version that was run does have the problematic file and is 11 commits out of date**.  I've tried wiping the build, but that doesn't seem to help.  I haven't updated the project settings in a while, so it's clearly getting a newer version than I pushed 6 months ago.  Any suggestions?

Finally, as I suspect this is a user error, rather than a readthedocs issue, is there a better place (e.g. a forum) to ask questions?

**Comment by agjohnson** [comment]
If the project is out of date, double check the Read the Docs webhook/service is established in your project settings. GitHub has had some issue yesterday and today, it's possible this might be a lingering issue.

**Comment by SteveDoyle2** [comment]
I found a reference to a the missing file in an RST file.  That presumably triggered Sphinx to import the pyc (maybe?) even though I wiped the build directory multiple times.   I pushed that new commit, which then forced Github to push a Read the Docs build (something that hadn't been working before Github went down), and the docs automatically build.  What's even stranger is Sphinx didn't have any problems with the missing file when I ran it on my local computer.

I'm not totally sure what happened, but it now works!  Thanks!
